packages/gas-fee-controller/src/GasFeeController.ts

```diff
@@ -348,9 +349,12 @@ export class GasFeeController extends StaticIntervalPollingController<
     this.getCurrentAccountEIP1559Compatibility =
       getCurrentAccountEIP1559Compatibility;
     this.#getProvider = getProvider;
-    this.EIP1559APIEndpoint = EIP1559APIEndpoint;
-    this.legacyAPIEndpoint = legacyAPIEndpoint;
+    this.EIP1559APIEndpoint = `${GAS_API_BASE_URL}/networks/<chain_id>/suggestedGasFees`;
+    this.legacyAPIEndpoint = `${GAS_API_BASE_URL}/networks/<chain_id>/gasPrices`;
     this.clientId = clientId;
+    this.infuraAuthToken = Buffer.from(
```

Comment:
Minor, but to avoid this logic living in the constructor, is it worth a `#buildInfuraToken` method?